### PR TITLE
Add enterprise_worker service user during provisioning.

### PR DESCRIPTION
**Pull request by douglashall**
Reverts edx/configuration#4022

This reintroduces the automated creation of the `enterprise_worker` user. The email address of the `enterprise_worker` user in stage/prod were different from what the original PR was specifying causing the previous rollout of this change to fail. I have now overridden the enterprise service worker user email address to match what is in stage/prod.

This PR depends on:
https://github.com/edx/edx-internal/pull/145
https://github.com/edx/edge-internal/pull/40

### CODE REVIEW

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -693,6 +693,7 @@ EDXAPP_BLOCK_STRUCTURES_SETTINGS:
 # Configuration settings needed for the LMS to communicate with the Enterprise service.
 EDXAPP_ENTERPRISE_API_URL: "{{ EDXAPP_LMS_ROOT_URL }}/enterprise/api/v1"
 
+EDXAPP_ENTERPRISE_SERVICE_WORKER_EMAIL: "enterprise_worker@example.com"
```
Comment (asadiqbal08) [inline playbooks/roles/edxapp/defaults/main.yml:696]: @douglashall one thing I want to know, Will that variable use the value from [edx-internal](https://github.com/edx/edx-internal/pull/145/files#diff-b8465b257bd194c38c12892a42f2c1b9R557) repo in a case of stage/prod?

Comment (douglashall) [reply playbooks/roles/edxapp/defaults/main.yml:696]: Yes, it will.

**Comment by douglashall** [comment]
@feanil @jibsheet Could one of you take a look at this PR when you get a chance?

**Review by feanil — APPROVED** [review]
Generally looks good, please add an entry to the changelog, explaining what and why you're doing this.

**Comment by douglashall** [comment]
@feanil Changelog added. Thanks.

**Review by feanil — APPROVED** [review]

**Pull request merged**